src/puppetlabs/puppetdb/http/nodes.clj

```diff
@@ -46,10 +47,12 @@
         (validate-query-params {}))}
```

Comment:
should we get the parent name in the node-status function above now too? These changes make our existing error inconsistent by not specifying "node"
```
curl -X GET http://localhost:8080/pdb/query/v4/nodes/foo
{
  "error" : "No information is known about foo"
}
```
vs
```
curl -X GET http://localhost:880/pdb/query/v4/nodes/foo/facts
{
  "error" : "No information is known about node foo"
}
```